### ISSUE TITLE
should the 'French site' default to french for everything?

### ISSUE THREAD

**Issue by annamorphism** [DDMAL/VIM#169]
I think it's a little weird that when you open the site and click on the French half of the site, the names come up in French but the site itself is still English ("More facets coming", "Instrument list", "showing XXX entries...")
I know we have these separate for a reason, but I don't think it would be obvious to a French user why only half the contents look translated. ...

<img width="587" alt="image" src="https://github.com/user-attachments/assets/4c12c2e3-532e-47a2-bd8e-143f50c7dcb0">

**Comment by dchiller** [comment]
I'm not sure whether this is doable given how things are implemented (I could look more) but I wonder if there's a way to have the "instrument name language" be set to the "site language" automatically where there is overlap, even though the content is coming from different places (google's api vs our database/wikidata)

**Comment by kunfang98927** [comment]
The solution now is:
- For "Site Language", we continue to use Google Translate, but block the instrument labels to avoid them being translated.
- For "Instrument Name Language", we will use the yellow drop-down menu to control the language.

**Comment by annamorphism** [comment]
actually come to think of it, if you ARE looking at the site in French already, the home page looks like this:
<img width="1131" alt="image" src="https://github.com/user-attachments/assets/e26292e4-1837-440a-9b04-d1f8fcd74c8e">
There's no way of knowing that the left "Commencer" takes you to a French site with English labels.
<img width="436" alt="image" src="https://github.com/user-attachments/assets/a1c8e358-a1d4-4c19-8876-f316a3121e5b">
but the right one gets you French with French labels: 
<img width="446" alt="image" src="https://github.com/user-attachments/assets/9c5eb0ba-fd05-4edf-af4c-0ef3000f7fce">

**Comment by kunfang98927** [comment]
Thank you for your comments! For the landing page, when clicking on "Get Started", we will set both ”site language“ and "instrument language" to English, and when clicking on "Commencer", we will set both ”site language“ and "instrument language" to French (Everything in the instrument-list page should then be in French).

Another issue mentioned is the design of landing page. Now my idea is to first block everything in the landing page to be translated by google translate except the two buttons. For the two buttons, **explicitly** point out whether we will be taken to a English/French site. For example, like this:
<img width="831" alt="image" src="https://github.com/user-attachments/assets/563ba957-18d7-4ea7-bc25-0ead28536bc7">

When I use google translate to translate the website to French, it will be like this:
<img width="824" alt="image" src="https://github.com/user-attachments/assets/60de2d88-f5c1-486b-a80e-1a0396d6e371">

When translating to Japanese, it will be like this:
<img width="818" alt="image" src="https://github.com/user-attachments/assets/25a5b382-bd0c-4578-8818-c25a773afe16">

**Comment by dchiller** [comment]
@kunfang98927 I think this idea of explicitly noting the language in button is a good one and will lead to less user confusion. 

My one continuing quibble with the design in those screenshots is that if I had my druthers I would like the main text (the one that says "where the past and present... etc.) to also be translated to the user's language. In other words, in your last example, I would like a user with Japanese browser language to be able read that paragraph in Japanese. I understand why we also want French and English to show up given our Canadian origin. And I like the idea of not just having a single text in the browser language because I suspect that a lot of times that would end up just being English by default, which is exactly the type of Anglo-centrism we are trying to avoid. Plus, if I want to see the Japanese names for instruments, I have to know to go to the English or French version of the site, and then in another drop-down select Japanese.

So here are some thoughts:

For the home page text, either (a) have the text a single time and have it translated to the browsers default language, or (b) choose more languages as defaults (eg. English and French, but also maybe Spanish, Mandarin, Turkish, and Hindi or any others for example) and put them all on the home page. And then for the button, have buttons in all those languages, or maybe have a button that is actually a dropdown that says "Visit Enlish site", "Visitez le site en francais",  etc. 

Basically, communicating with more options on the homepage that there are so many versions of the site, and not just two!

**Comment by kunfang98927** [comment]
I noticed that MIMO uses this method to achieve multi-language switching on the home page. Perhaps we can also adopt a similar approach.
<img width="878" alt="image" src="https://github.com/user-attachments/assets/84ada603-ddc8-498b-8ddb-c40ec6992da0" />
